doc/build.md

```diff
@@ -2,121 +2,193 @@
+
+***NOTE:*** Make sure that you don't have older USD locations in your ```PATH``` and ```PYTHONPATH``` environment settings. ```PATH``` and ```PYTHONPATH``` are automatically adjusted inside the project to point to the correct USD location. See ```cmake/usd.cmake```.
+
+#### 3. Download Universal Front End (UFE) 
+
+The Universal Front End (UFE) is a DCC-agnostic component that allows Maya to browse and edit data in multiple data models. This allows Maya to edit pipeline data such as USD.
+UFE is developed as a separate binary component, and is therefore versioned separately from Maya. UFE is installed as a built-in component of Maya.
+
+UFE headers and Doxygen documentation is shipped with Maya Devkit and can be downloaded from the link below: 
+
+https://www.autodesk.com/developer-network/platform-technologies/maya  
+
+***NOTE:*** UFE is only supported for Maya 2019 and later. Earlier versions of Maya should still be able to load the plug-ins but without the features enabled by UFE.
+
+#### 4. Download the source code
 
 Start by cloning the repository:
 ```
-git clone <git_path_to_maya-usd>
+git clone https://github.com/Autodesk/maya-usd.git
 cd maya-usd
 ```
-#### 3. Script arguments
+#### 5. How To Use build.py Script
+
+##### Arguments
 
 There are four arguments that must be passed to the script: 
 
-- --maya-location   ---> Directory where Maya is installed.
-- --pxrusd-location ---> Directory where Pixar USD Core is installed.
-- --devkit-location ---> Directory where Maya Devkit is installed.
-- workspace_location  ---> Directory where the project use as a workspace to build and install plugin/libraries.
+| Flags                 | Description                                                                           |
+|--------------------   |-------------------------------------------------------------------------------------- |
+|   --maya-location     | directory where Maya is installed.                                                    |
+|  --pxrusd-location    | directory where Pixar USD Core is installed.                                          |
+|  --devkit-location    | directory where Maya devkit is installed.                                             |
+| workspace_location    | directory where the project use as a workspace to build and install plugin/libraries  |
 
-Comma-separated list of cmake variables can be also passed to build system using ```--build-args```.
-e.g
 ```
-python build.py --build-args="-DBUILD_ADSK_PLUGIN=ON,-DBUILD_PXR_PLUGIN=OFF"
+➜ maya-usd python build.py --maya-location /usr/autodesk/maya2020 --pxrusd-location /usr/local/USD-Master /usr/local/workspace
+➜ maya-usd python build.py --maya-location /Applications/Autodesk/maya2020 --pxrusd-location /opt/local/USD-Master /opt/local/workspace
+c:\maya-usd> python build.py --maya-location "C:\Program Files\Autodesk\maya2020" --pxrusd-location C:\USD-Master C:\workspace
```

Comment:
The default Maya install location folder is with an uppercase `M` for `Maya2020`. Does that matter?